src/Server/TaskHandlerChain.php

```diff
@@ -31,20 +34,15 @@ class TaskHandlerChain
      *  - Auth next so that spam calls can't DDOS through the mutex implementation
      *  - Then mutex so that a given task is only processed once at a time
      *
-     * @param \Ingenerator\CloudTasksWrapper\Server\Middleware\TaskLoggingMiddleware               $logging
-     * @param \Ingenerator\CloudTasksWrapper\Server\Middleware\ExceptionCatchingMiddleware         $catcher
-     * @param \Ingenerator\CloudTasksWrapper\Server\Middleware\TaskRequestAuthenticatingMiddleware $auth
-     * @param \Ingenerator\CloudTasksWrapper\Server\Middleware\TaskMutexLockingMiddleware          $mutex
-     *
-     * @return \Ingenerator\CloudTasksWrapper\Server\TaskHandlerChain
      */
     public static function makeDefault(
         TaskLoggingMiddleware $logging,
         ExceptionCatchingMiddleware $catcher,
         TaskRequestAuthenticatingMiddleware $auth,
-        TaskMutexLockingMiddleware $mutex
+        TaskMutexLockingMiddleware $mutex,
+        JsonBodyParsingMiddleware $json_parsing
```

Comment:
I guess we'll make this a `0.4` so it's a breaking release?

Reply:
yeah, going to release it with #7 too